### PR TITLE
DDP-6078, DDP-6091, DDP-6086

### PR DESCRIPTION
**Pull request by YuriyPavlovBI**
Some RARE-X fixes

### CODE REVIEW

File: study-builder/studies/rarex/i18n/en.json [hunk 1]
```diff
@@ -191,15 +191,19 @@
...
   "general_information_gender_identity_man": "Man",
   "general_information_gender_identity_woman": "Woman",
+  "general_information_gender_identity_boy": "Boy",
+  "general_information_gender_identity_girl": "Girl",
   "general_information_gender_identity_nonbinary": "Nonbinary",
   "general_information_gender_identity_transgender": "Transgender",
   "general_information_gender_identity_none_of_above": "None of these describe me, and I’d like to describe myself",
   "general_information_gender_prefer_not_to_answer": "Prefer not to answer",
+  "general_information_gender_other_detail": "Other gender specify",
+  "general_information_gender_other_detail_req_hint": "Please specify other gender",
```
Comment (yufengwng) [inline study-builder/studies/rarex/i18n/en.json:206]: I think we should add `*` to indicate that it's required.
```suggestion
  "general_information_gender_other_detail_req_hint": "Please specify other gender *",
```

File: study-builder/studies/rarex/general-information.conf [hunk 2]
```diff
@@ -2098,20 +2098,199 @@
...
+                          "text": ${i18n.en.general_information_gender_other_detail_req_hint}
+                      }
+                      ]
+                    }
+                  ]
+                }
+              },
+              {
+                "ruleType": "LENGTH",
+                "maxLength": 50,
```
Comment (yufengwng) [inline study-builder/studies/rarex/general-information.conf]: Let's make this 500. See Slack [here](https://broadinstitute.slack.com/archives/C01A6HWK3PX/p1620833065217400?thread_ts=1620832797.216900&cid=C01A6HWK3PX). Can you also update the other limits, such as for the "other detail" for "race" question? Thanks!

File: study-builder/studies/rarex/study.conf [hunk 3]
```diff
@@ -2550,7 +2552,8 @@
         ]
       },
       "preconditionExpr": """!user.studies["rarex"].forms["GENERAL_INFORMATION"].questions["PARTICIPANT_FIRST_NAME"].isAnswered()
-        && user.studies["rarex"].forms["PARENTAL_CONSENT"].hasInstance()""",
+        && user.studies["rarex"].forms["PARENTAL_CONSENT"].hasInstance()
+        && !user.studies["rarex"].forms["GENERAL_INFORMATION"].instances[latest].hasPreviousInstance()""",
```
Comment (yufengwng) [inline study-builder/studies/rarex/study.conf]: I've added copy events for the other 2 LAR consents. Can you update your branch and update those too? Thanks!

Comment (YuriyPavlovBI) [reply study-builder/studies/rarex/study.conf]: Sure!

File: study-builder/studies/rarex/general-information.conf (hunk 2 above)
Comment (YuriyPavlovBI) [reply study-builder/studies/rarex/general-information.conf]: Done

File: study-builder/studies/rarex/i18n/en.json (hunk 1 above)
Comment (YuriyPavlovBI) [reply study-builder/studies/rarex/i18n/en.json:206]: It's a validation message. I'll add * to the placeholder

File: study-builder/studies/rarex/i18n/en.json [hunk 4]
```diff
@@ -28,7 +28,7 @@
   "diagnosis_other": "Other",
 
   "add_participant_name": "Add Participant",
-  "add_participant_prompt_relationship": "Choose option that best describes your relationship with the patient",
+  "add_participant_prompt_relationship": "Please specify your relationship with the patient",
```
Comment (yufengwng) [inline study-builder/studies/rarex/i18n/en.json]: Bad merge. The one that says `Choose option ...` is the correct one.

**Comment by yufengwng** [comment]
Yuriy is offline, so I'm going to move the ball forward here.

**Review by yufengwng — APPROVED** [review]

**Pull request merged**